account_payment_method_fs_storage/models/account_payment_method.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountPaymentMethod(models.Model):
+
+    _inherit = "account.payment.method"
+
+    storage_id = fields.Many2one(
```

Comment:
I have my doubts on this point. What are the access rights on this model? Accountants will be able to select any record. Not sure that's a good thing. I think it would be better to define on a storage that it can be used in this case and filter the result.

Reply:
My proposition:

- Introduce a boolean on fs.storage that tells if storage can be used on Payment Method
- Do not give access to fs.storage but run in sudo the push operation to storage
- Use a selection "storage" that represents the list of storage that can be used on Payment Method